pyproject.toml

```diff
@@ -72,6 +72,7 @@ tutorials = [
     "matplotlib",
     "datasets>=2.8.0",
     "scikit-learn>=1.1.1",
+    "ipython<=8.12",
```

Comment:
"ipython<=8.12; python_version < '3.9'"

Reply:
probably the way to do it

Reply:
what, didn't know you could do that. nice!

Reply:
I stole it from spacys requirements ;)